apps/shared/avifutil.h

```diff
@@ -21,6 +21,7 @@
 #endif
 
 void avifImageDump(avifImage * avif);
+void avifContainerDump(avifDecoder * decoder);
```

Comment:
Joe: Please let me know if you would prefer a different name for the new function.